blog/_posts/2018-11-18-RGSoCTeamsGOTOBerlin.md

```diff
@@ -0,0 +1,64 @@
+---
+title: RGSoC Teams at GOTO Berlin!
+layout: post
+created_at: Tue Dec 18 2018
+permalink: blog/2018-11-18-RGSoCTeamsGOTOBerlin
```

Comment:
please change the date in both permalink and post title to 2018-12-18 so that it shows up as the last post :)